src/main/java/com/sarahhand/fractals/view/FractalsUI.java

```diff
@@ -241,47 +234,26 @@ public void actionPerformed(ActionEvent ae) {
 	 * @author M00031
 	 *
 	 */
-	private class LoadMandelbrotMenuItemActionListener implements ActionListener {
+	private class LoadMenuItemActionListener implements ActionListener {
 
 		private ConfigSaverLoader saverLoader = ConfigSaverLoader.getDefaultConfigSaverLoader();
 		private JFileChooser fileChooser = new JFileChooser();
 
 		public void actionPerformed(ActionEvent ae) {
 			if(fileChooser.showDialog(null, "Load") == JFileChooser.APPROVE_OPTION) {
-				mandelbrotEvents.getFractalViewer().setConfig((FractalConfig)saverLoader.load(
-						mandelbrotEvents.getFractalViewer().getConfig().getClass(),
-						fileChooser.getSelectedFile().getAbsolutePath()));
+				FractalConfig newConfig = saverLoader.load(fileChooser.getSelectedFile().getName());
+				if(newConfig instanceof MandelbrotConfig) {
```

Comment:
Can you get rid of these System.out.println() statements? Either replace it with log.debug() statements, or remove it entirely (including the if/then/else).